src/main/main.js

```diff
@@ -143,7 +143,18 @@ export function main(argv) {
     }
   }
 
-  app.on('ready', createWindow);
+  app.on('ready', async () => {
```

Comment:
I'm not entirely sure how this works, but I can at least tell it's a lot simpler design than what we had for setting up dev mode.

Reply:
This is just for loading the react-devtools extension in the browser. It follows that packages documentation about when exactly do that, relative to actually creating the BrowserWindow.